site/docs/components/navigation-item/accessibility.mdx

```diff
@@ -0,0 +1,67 @@
+---
```

Comment:
This page is throwing a 500 error in the preview, there might be an issue with the mdx